### ISSUE TITLE
Angular Error: `req is not defined`

### ISSUE THREAD

**Issue by Codehhh** [tidusjar/Ombi#1683]
#### Ombi build Version:

V 3.0.2314-PR

#### Update Branch:

Open Beta

#### Media Sever:

NA

#### Media Server Version:

NA

#### Operating System:

UnRaid Docker

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```
2017-11-05 12:47:02.082 -05:00 [Information] Start installing Hangfire SQL objects...
2017-11-05 12:47:06.390 -05:00 [Information] Hangfire SQL objects installed.
2017-11-05 12:47:06.439 -05:00 [Information] Starting Hangfire Server
2017-11-05 12:47:06.446 -05:00 [Information] Using job storage: 'SQLite Server: /config/Ombi.db'
2017-11-05 12:47:06.446 -05:00 [Information] Using the following options for Hangfire Server:
2017-11-05 12:47:06.446 -05:00 [Information]     Worker count: 1
2017-11-05 12:47:06.447 -05:00 [Information]     Listening queues: 'default'
2017-11-05 12:47:06.447 -05:00 [Information]     Shutdown timeout: 1.00:00:00
2017-11-05 12:47:06.447 -05:00 [Information]     Schedule polling interval: 00:00:15
```

#### Problem Description:

![image](https://user-images.githubusercontent.com/12055335/32417399-34f00332-c227-11e7-8d49-bf7494aeb556.png)

#### Reproduction Steps:

Pull fresh docker and start.

**Comment by tidusjar** [comment]
That happens on the first page?

**Comment by Codehhh** [comment]
Yes, before initial setup

**Comment by weirdcrap** [comment]
I am also experiencing this issue.

**Comment by daymanbpi** [comment]
pulled fresh docker. latest. 
same. webpage is just stuck on loading... screen.

**Comment by that1guy** [comment]
Confirmed.  Same issue here.

**Comment by tidusjar** [comment]
Looks like this is only happening in the PR builds... How is everyone updating?

**Comment by tidusjar** [comment]
Ok i'll speak to the lsio team.

I've currently disabled pull requests from building on AppVeyor so that should prevent this until it's fixed in the container.

I've kicked off a new build so if you guys update it should now be fine.

**Comment by tidusjar** [comment]
Nope, looks like it's still an issue

**Comment by tidusjar** [comment]
https://github.com/angular/angular-cli/issues/8359

**Comment by weirdcrap** [comment]
After a fresh docker pull mine now seems to be working...

**Comment by gilbN** [comment]
Working for me too

**Comment by Codehhh** [comment]
tidusjar created a workaround for the time being as it's an external error
with angular 5.

**Comment by Codehhh** [comment]
@tidusjar the angular-cli issue is closed, just need to update uglify-es to 3.1.8

**Comment by PotatoQuality** [comment]
@Codehhh thanks for the headsup. 
I saw your code push on this one @tidusjar . Can we close this issue?
